easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,37 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Pau Ruiz <pau.ruiz@hpcnow.com>, Arturo Gimeno
+# License::     GPL-v3.0
+# #
+
+easyblock = 'MakeCp'
+
+name = 'FragGeneScanPlus'
+version = '20161116'
+
+homepage = 'https://github.com/hallamlab/FragGeneScanPlus'
+description = """FragGeneScan-Plus for scalable high-throughput short-read open reading frame prediction"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/hallamlab',
+        'repo_name': 'FragGeneScanPlus',
+        'commit': '91b0ab6',
+    },
+}]
+checksums = ['53d88b435eeb6b32e64a7be189a8903fa23bcb22694d7a322d564490e9cd3312']
+
```

Comment:
As the bot indicates, this should have a `binutils` build dependency.

Reply:
hi @casparvl i already fixed that! can you check? thanks

Reply:
This part is fine, CI passes now